### DISCUSSION TITLE
vlc as default player for next release

### DISCUSSION THREAD

**Discussion by perfectsyrup** [Joshua-Riek/ubuntu-rockchip#430] [Ideas]
VLC player build from this ppa uses much less CPU for 4K high bitrate video than default mpv player and it has good multithreading on all cores. Mpv usually have very high 2 cores usage.
https://launchpad.net/~liujianfeng1994/+archive/ubuntu/rockchip-multimedia/+sourcepub/15192443/+listing-archive-extra 
I have switched to vlc and now RK3588 really shines as a video player.

Suggest to make vlc as a default player for 1.29 build and preinstall it in OS image.

ps. "sudo apt-get install vlc=1:3.0.18-4" to install it right now.

**Comment by watou** [comment]
I installed the referenced vlc without having to choose a PPA or specifying a particular version. Was that different from your experience?

I get about 20% frame loss on a 4K H.264 AVC1 video at 24fps on my Rock5B.  Do you get better performance? I assume performance will improve as video optimisations are improved for the RK3588.

**Comment by Joshua-Riek** [comment]
I need to publish a new release, which is building now, so I won't get to this. However, I have found MPV to work better for most video formats over VLC. There may be some new changes I don't know about to improve VLC performance and compatibility.

**Comment by cantalupo555** [comment]
I prefer SMPlayer or MPV.
It have a better performance.
